cloud/datastore/src/sorting/Sort.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Sorting directions
+ */
+export enum DIRECTION {
+  ASC,
+  DESC
+}
+
+
+/**
+ * Interface used to monitor sorting requirements
+ */
+export interface SortOrder {
+  // List of fields to sort results
+  sortData: string[]
+  // Sort direction
+  direction: DIRECTION
```

Comment:
probaby a type `'ASC' | 'DESC'` fits better here than an enum, the ts implementation is quite heavyweight.

https://www.typescriptlang.org/docs/handbook/enums.html (see 'compiled to:')